application/ui/src/app/pages/matching-result/matching-result.page.html

```diff
@@ -57,3 +59,220 @@ <h3>{{user.name}}</h3>
+            <ion-list>
+
+              <ion-item>
+                <ion-avatar slot="start">
+                  <ion-img src="https://i.pravatar.cc/300?u=a"></ion-img>
+                </ion-avatar>
+                <ion-label>
+                  <h2>Frank Tchatseu</h2>
+                </ion-label>
+              </ion-item>
+            </ion-list>
+          </ion-col>
+          <ion-col>
+            <ion-list>
+              <ion-item>
+                <ion-avatar slot="start">
+                  <ion-img src="https://i.pravatar.cc/300?u=a"></ion-img>
+                </ion-avatar>
+                <ion-label>
+                  <h2>Urbain</h2>
+                </ion-label>
+              </ion-item>
+
+            </ion-list>
+          </ion-col>
+        </ion-row>
+        <ion-row>
+          <ion-col col-md-8>
+            <ion-item button="true" id="open-date-input">
+              <ion-label> Date de rencontre </ion-label>
+              <ion-text slot="end">{{ conferenceDate | date: 'mediumDate' }}</ion-text>
+              <ion-popover id="date-input-popover" trigger="open-date-input" [showBackdrop]="false" side="top" alignment="end">
+                <ng-template>
+                  <ion-datetime max="2056" [(ngModel)]="conferenceDate" presentation="date"></ion-datetime>
+                </ng-template>
+              </ion-popover>
+            </ion-item>
+          </ion-col>
+          <ion-col col-md-4>
+            <ion-item button="true" id="open-time-input">
+              <ion-label> Heure </ion-label>
+              <ion-text slot="end">{{ time}}</ion-text>
+              <ion-popover id="date-input-popover" trigger="open-time-input" [showBackdrop]="false" side="top" alignment="end">
+                <ng-template>
+                  <ion-datetime [(ngModel)]="time" presentation="time"></ion-datetime>
+                </ng-template>
+              </ion-popover>
+            </ion-item>
+          </ion-col>
+        </ion-row>
+        <ion-list>
+          <ion-item>
+            <ion-label>Jour de rencontre</ion-label>
+            <ion-select multiple="true" [(ngModel)]="selectedDays">
+              <ion-select-option [value]="day" *ngFor="let day of allDays">{{day}}</ion-select-option>
+            </ion-select>
+          </ion-item>
+
+        </ion-list>
+        <br />
+        <ion-button color="primary" fill="clear">
+          Envoyer l'email
+          <ion-icon slot="end" name="mail"></ion-icon>
+        </ion-button>
+      </ion-content>
+    </ng-template>
+  </ion-modal>
+</ion-content>
+
+<ion-content class="ion-padding">
+  <ion-modal #modal trigger="open-modal2">
+    <ng-template>
+      <ion-content>
+        <ion-toolbar>
+          <ion-title>Période de matching</ion-title>
+          <ion-buttons slot="end">
+            <ion-button color="light" (click)="modal.dismiss()">Fermer</ion-button>
+          </ion-buttons>
+        </ion-toolbar>
+        <ion-list style="max-width:230px">
```

Comment:
do not add style in html use a class and put that in scss moreover use rem not px with 1rem=16px